src/__tests__/eventListenerCallbacks.spec.ts

```diff
@@ -277,6 +276,7 @@ describe('react-native-community/netinfo', () => {
                   cellularGeneration: 'unknown',
                 },
               },
+              expectedIsInternetReachable: null,
```

Comment:
is this tri-state (false/true/null) vs just boolean (false/true)?

Reply:
It is a tri-state.